server/game/cards/03-WC/AEmbertracker.js

```diff
@@ -0,0 +1,19 @@
+const Card = require('../../Card.js');
+
+class AEmbertracker extends Card {
```

Comment:
We usually skip the E in cards that have "amber" in their name

Reply:
resolved. will require WC.json update as welll.